package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "nodeunit"
+    "test": "jshint bin/ lib/ test/ *.js && nodeunit"
```

Comment:
if its just white space just run jscs with fix and do it as part of this its a small enough repo.